solarwinds_apm/apm_logging.py

```diff
@@ -38,13 +38,46 @@
 import os
 
 
+class ApmLoggingType:
+    """Mapping of solarwinds_apm library log types"""
+
+    log_types = {
+        "STDERR": 0,
+        "STDOUT": 1,
+        "FILE": 2,
+        "NULL": 3,
+        "DISABLED": 4,
+    }
+
+    @classmethod
+    def default_level(cls):
+        """Returns integer representation of default log type"""
+        return cls.log_types["STDERR"]
+
+    @classmethod
+    def disabled_level(cls):
+        """Returns integer representation of disabled log type"""
+        return cls.log_types["DISABLED"]
+
+    @classmethod
+    def is_valid_level(cls, level):
+        """Returns True if the provided level is a valid interger representation of log type, False otherwise."""
+        try:
+            level = int(level)
+            return bool(level in list(cls.log_types.values()))
+        except (ValueError, TypeError):
+            return False
```

Comment:
Wouldn't these make more sense as `default_type`/`disabled_type`/`is_valid_type` ?

Reply:
Yes! Changed in [62624db](https://github.com/solarwinds/apm-python/pull/259/commits/62624db411743d7443e6f07d6165e1e9d49a004d)